### PR TITLE
Provide a background

### PR DESCRIPTION
**Pull request by AlanGriffiths**
Add a simple background

### CODE REVIEW

File: src/ubuntu_core_kiosk_main.cpp [hunk 1]
```diff
@@ -20,22 +20,56 @@
...
+    ExternalClientLauncher launcher;
+
+    auto background = "#1f1f1f"s;
+
+    auto startup_background = [&](std::string const& swaybg_launch_path)
+        {
+            launcher.launch({swaybg_launch_path, "-c", background});
+        };
+
+    auto const default_swaybg_launch_path = []
```
Comment (wmww) [inline src/ubuntu_core_kiosk_main.cpp]: Does this need to be a lambda? It seems it's being executed unconditionally just below. Why not do the logic here and set a variable with the result?

**Review by wmww — CHANGES_REQUESTED** [review]

File: src/ubuntu_core_kiosk_main.cpp (hunk 1 above)
Comment (AlanGriffiths) [reply src/ubuntu_core_kiosk_main.cpp]: Fixed

Comment (wmww) [reply src/ubuntu_core_kiosk_main.cpp]: I'm not crazy about using lamdas for control flow. Discussion in matermost.

Comment (RAOF) [reply src/ubuntu_core_kiosk_main.cpp]: In this case, right now, I don't think the immediately-executed-lambda is particularly valuable; it is immediately used and then just as immediately falls out of scope, so the benefit of `const`ing the variable is limited.

I don't *object* to using an IEL, though.

**Review by RAOF — APPROVED** [review]
I'm ok with this.

File: src/ubuntu_core_kiosk_main.cpp (hunk 1 above)
Comment (AlanGriffiths) [reply src/ubuntu_core_kiosk_main.cpp]: Fixed more

**Review by wmww — APPROVED** [review]
LGTM

bors r+

**Comment by AlanGriffiths** [comment]
> Configuration problem:
> bors.toml: not found

Now that's another PR we need!

**Pull request merged**